plugins/Contents/API.php

```diff
@@ -57,8 +57,6 @@ private function getDataTable($name, $idSite, $period, $date, $segment, $expande
      */
     private function filterDataTable($dataTable)
     {
-        $dataTable->filter('Sort', array(Metrics::INDEX_NB_VISITS));
-
```

Comment:
No longer needed since we do now always sort in GenericFilter. So we do no longer sort the entire table twice